advanced-tools/index.rst

```diff
@@ -0,0 +1,10 @@
+================
+ Advanced Tools
+================
+
+.. toctree::
+   :maxdepth: 1
```

Comment:
Since the index pages are mostly empty, wouldn't it be better to add more depth?
This will provide a better overview of the pages contents (since it might not be obvious which page contains what you are looking for), and provide direct links to specific sections.